internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -106,6 +109,15 @@ func (*Sequencer) Initialize(r runtime.Runtime) []runtime.Phase {
 			"earlyServices",
 			StartUdevd,
 			StartMachined,
+		).AppendWithDeferredCheck(
+			func() bool {
+				disabledStr := procfs.ProcCmdline().Get(constants.KernelParamDashboardDisabled).First()
```

Comment:
we need a note in `hack/release.toml` that now we switch to tty2, and that logs run on tty1, and that dashboard can be disabled by doing this and that

in the docs, kernel param reference, we should add new kernel arg

Reply:
👍 done, also disabled it by default on SBCs.